discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown

```diff
@@ -278,7 +280,15 @@ Settings* &rarr; *Foundation*.
 *Hosts allowed* field.    
     d) Click *Update*.
     
-You should proceed only when all three major steps have been completed. 
+4. The same users must exist on both remote and the staging environments. In
+addition, the users must have the same credentials (that is, same screen name,
+email, and roles, password).  
+
+**Important note:** do not simply clone the database or else the Remote
+Publication process will also fail.
+
+
+You should proceed only when all four major steps have been completed. 
 
```

Comment:
Yeah, I'll verify this with Máté. He may also have some additional info related to this too. I'll let you know.